test/ansible/run_test.yaml

```diff
@@ -146,6 +146,15 @@
   - name: Debug tests output
     debug: "msg={{ loadbalancerip.stdout }}"
 
+  - name: prepare infrastructure-operator on target
+    shell: |
+      envsubst < "{{ src_dir }}/test/e2e/manifests/infrastructure-operator/kustomization.yaml.tpl" > "{{ remote_manifests_path }}/infrastructure-operator/kustomization.yaml"
```

Comment:
Why output in `{{ remote_manifests_path }}`? BTW I think this might be unused, we can remove it